python/hawkey/tests/tests/test_reldep.py

```diff
@@ -35,20 +35,15 @@ def test_basic(self):
         reldep = requires[0]
         self.assertEqual(str(reldep), "P-lib >= 3")
 
-    def test_unicode(self):
```

Comment:
Originally it raises hawkey.ValueException, but now it is quite happy with this string, but it cannot find any result that correspond to it. The question is if this behavior is correct and helpful for debugging.